src/main.nr

```diff
@@ -167,21 +168,39 @@ contract PrivateOracle {
         let address_this = AztecAddress::new(context.this_address());
         let mut escrow_randomness = 0;
 
+ 
+
         if(payment_token.address != 0) {
```

Comment:
Given that this is immutable, can't this be checked at construction time and avoid this check altogether? If it's not `0` at construction, and it can't be changed, then it will never be 0, right?

Reply:
Dayum. Great to know!